### ISSUE TITLE
Implement prepare feature for buildroot repository enablement

### ISSUE THREAD

**Issue by thrix** [teemtee/tmt#2678]
Similar to epel and crb.

Crosslink: https://issues.redhat.com/browse/TFT-1617

**Comment by thrix** [comment]
@idorax could you maybe also this one, seems the same as CRB and EPEL?

**Comment by idorax** [comment]
Sure, @thrix, let me work on it, thanks!

**Comment by thrix** [comment]
> Sure, @thrix, let me work on it, thanks!

thank you!

**Comment by falconizmi** [comment]
Hi @idorax , I would like to continue with this issue if you don't mind. Do you still want to contribute to this?

**Comment by thrix** [comment]
@falconizmi I do not think so, he left RH

**Comment by psss** [comment]
@falconizmi, thanks for interest in implementing this! What do you think could be a realistic time frame? Shall we try targetting `1.37` in September? Or rather the next release?

**Comment by falconizmi** [comment]
> @falconizmi, thanks for interest in implementing this! What do you think could be a realistic time frame? Shall we try targetting `1.37` in September? Or rather the next release?

The implemention is done but it hasnt been tested yet. I would add it as a topic to the meeting for the next week where I would ask people, if they have some tests for buildroot, to test it for me.

**Comment by falconizmi** [comment]
@CtrlZmaster there is in internal tmt repo script to enable rhel buildroot. If you want to run the script you can run command like this: `tmt run provision -h connect -g 10.31.44.95 -u root -k /home/iquwarah/.ssh/auto-qe prepare --insert --name rhel-buildroot --order 20 --how shell --script "wget SCRIPT_LINK && sh rhel-buildroot.sh"` just replace the SCRIPT_LINK with link to the internal script (as raw)

**Comment by psss** [comment]
This has been escalated by @brakogh as an important issue for the compose qualification testing. Proposing the highest priority. @falconizmi, will you have time to work on this during the next two weeks? Could we possibly include it in the next `tmt` release?

**Comment by falconizmi** [comment]
Okay, I will work on it and try best to make it to the next release

**Comment by psss** [comment]
Great, thanks! We're trying to finish `priority | must` features with some time reserve (one week before the release, which was yesterday 😁), but let's try to still make it! 👍

**Comment by bajertom** [comment]
Hi @falconizmi, any update on this issue? Is it going to make it into 1.46?

**Comment by falconizmi** [comment]
@bajertom sorry, I was sick these past days, so it wont make it into 1.46

**Comment by psss** [comment]
Actually, @falconizmi has already created an internal pull request with the plugin code:

* baseos-qe/tmt/-/merge_requests/106

Currently waiting for review.

**Comment by happz** [comment]
@falconizmi shall we close this issue as resolved, now when the downstream MR has been merged?

**Comment by falconizmi** [comment]
@happz yes, buildroot is currently working for rhel. The MR has been merged, but I still need to add a patch to allow only rhel and throw an exception for other images. However, this is not related to this issue, so we can close it.
